Examples/Neutral_density_calculation.ipynb

```diff
@@ -0,0 +1,790 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    t, s = load_temp_salt('mom5')
</pre><p>Can we use T, S for temperature and salinity? t is often used for time....</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/417/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>